src/components/AssetIcon.tsx

```diff
@@ -784,19 +784,25 @@ export const AssetIcon = ({
   className?: string;
 }) =>
   isAssetSymbol(symbol) ? (
-    <img
-      src={assetIcons[symbol]}
-      className={className}
-      alt={symbol}
-      tw="h-[1em] w-auto rounded-[50%]"
-    />
+    <$AssetIcon src={assetIcons[symbol]} className={className} alt={symbol} />
   ) : (
     <Placeholder className={className} symbol={symbol ?? ''} />
   );
+
+const $AssetIcon = styled.img`
+  --asset-icon-height: 1em;
+
+  height: var(--asset-icon-height);
+  width: auto;
+  border-radius: 50%;
+`;
+
 const $Placeholder = styled.div`
+  --asset-icon-height: 1em;
+
   background-color: var(--color-layer-5);
-  width: 1em;
-  height: 1em;
+  width: var(--asset-icon-height);
+  height: var(--asset-icon-height);
```

Comment:
I imagine theres a couple other places that rely on `font-size` to adjust the width/height. Were you able to audit all the usages to use `--asset-icon-height` instead?

Reply:
maybe we should also slap a `min-width`/`min-height` as well?

Reply:
hmm `--asset-icon-height` defaults to `1em` on line 801, so if I'm understanding correctly, this shouldn't actually impact other existing usages should it? 👀 (or maybe I'm misunderstanding) - but def agree on min-width/height! will add

Reply:
ah actually did u mean this [fix](https://github.com/dydxprotocol/v4-web/pull/1098) I ended up having to do 😢